pkg/controller/servicebindingrequest/sbrcontroller_test.go

```diff
@@ -121,55 +173,53 @@ func TestSBRControllerBuildGVKPredicate(t *testing.T) {
 			wanted bool
 			a      runtime.Object
 			b      runtime.Object
-			metaA  metav1.Object
-			metaB  metav1.Object
 		}{
 			{
 				name:   "non supported update",
 				wanted: false,
 				a:      deploymentA,
 				b:      deploymentA,
-				metaA:  deploymentA.GetObjectMeta(),
```

Comment:
why changing test fixtures on the existing tests?

Reply:
Answered above